case-lib/lib.sh

```diff
@@ -293,6 +293,29 @@ find_ldc_file()
     printf '%s' "$ldcFile"
 }
 
+func_cavstool_etrace_collect()
```

Comment:
Add a comment / header documenting important control flow expectations
- whether this function leaves anything running in the background. Any timeout.
- what the caller is expected to clean up and when